engines/e_afalg.c

```diff
@@ -817,11 +843,20 @@ static int afalg_finish(ENGINE *e)
     return 1;
 }
 
+static int free_cbc(void)
+{
+    short int i;
+    for(i=0; i < sizeof(afalg_cipher_nids)/sizeof(afalg_cipher_nids[0]);i++) {
```

Comment:
As above

Reply:
Hi i dont understen , how i help u an repair this

Reply:
Ayuda me quieren robar mi cuenta